executor-service/src/test/java/executor/service/service/webDriver/impl/WebDriverInitializerTest.java

```diff
@@ -3,25 +3,35 @@
 import executor.service.model.ProxyConfigHolder;
 import executor.service.model.ProxyCredentials;
 import executor.service.model.ProxyNetworkConfig;
-import executor.service.service.ConfigPropertiesLoader;
-import executor.service.service.webDriver.WebDriverInitializer;
+import executor.service.model.WebDriverConfig;
 import io.github.bonigarcia.wdm.WebDriverManager;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 
 import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.when;
 
 public class WebDriverInitializerTest {

```

Comment:
Remove this 'public' modifier: JUnit5 test classes and methods should have default package visibility